web/docs/deploying.md

```diff
@@ -8,7 +8,50 @@ Wasp is in beta, so keep in mind there might be some kinks / bugs, and possibly
 If you encounter any issues, reach out to us on [Discord](https://discord.gg/rzdnErX) and we will make sure to help you out!
 :::
 
-Right now, deploying of Wasp project is done by generating the code and then deploying generated code "manually", as explained below.
+# Automated
+
+## Wasp CLI
+
+Using the Wasp CLI, you can easily deploy your app to [Fly.io](https://fly.io) (which will be within their [free allowances](https://fly.io/docs/about/pricing/#free-allowances) if this is your only app) with just three commands:
```

Comment:
Make a note to the user they may want to add their CC info if they already have any Fly.io projects. Ref: https://www.notion.so/wasp-lang/Fly-Deploy-comments-61de123caf2348aabb2ccd6d972c84ff